### DISCUSSION TITLE
Ensure that the data_description name and data_asset name match

### DISCUSSION THREAD

**Discussion by jtyoung84** [AllenNeuralDynamics/aind-data-schema#1080] [Ideas]
There seems to be a common issue with the name in the data_description.json files not matching the s3_prefix. It mostly stems for those names being generated by two separate processes. It might be helpful to have some guidelines. Since it's much more difficult to rename folders in s3, those names should be considered canon, in my opinion.

**Comment by saskiad** [comment]
do you mean guidelines on what the names are? Or something that assures that the names match? We have the former.

**Comment by jtyoung84** [comment]
I think it's more for existing data_description files. If the names in the data_description file don't match the folder in S3, there are two options:

1) Replace the name in the data_description file to match the s3 location (which is easy)
2) Rename the folder in S3 to match the name in the data_description file (which is harder)
